src/button/button-group.scss

```diff
@@ -7,37 +7,29 @@
   align-items: center;
   isolation: isolate;
 
-  $childSelector: 'input, button'; // target nested inputs and buttons
+  $childZindexSelector: '.iui-input-container, .iui-button, .iui-input:where(:not(.iui-input-container .iui-input))';
```

Comment:
Can you explain or paste a link to explaination, how this is working `.iui-input:where(:not(.iui-input-container .iui-input))`? Because I can't find why this magic is working 😀

Reply:
> Can you explain or paste a link to explaination

What part is not clear? It is targeting `iui-input` only if it is NOT a descendant of `iui-input-container`. And it is also targeting `iui-input-container` separately.

> specificity of this one is really big

Anything inside [`:where`](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) does not count towards specificity, so that part is fine (the resulting specificity is 0-3-0).

The only place where it has huge specificity is for active stripe. I don't know what we can do about that.
![image](https://user-images.githubusercontent.com/9084735/152867364-d3e02a74-cdb0-4d47-988b-9e0fb222c0c7.png)

Reply:
stylelint should pass now

Reply:
Most of selectors now how 0-3-0 and there is a bunch of 0-4-0
![image](https://user-images.githubusercontent.com/81580355/153374272-6e2c37ed-0ade-4d84-97b6-908c1a63461c.png)

Do no remember, what is the specificity we are trying to achieve in most places.

Reply:
ok yes forgot about that `borderless` selector, so it will be 0-4-0 in a few places. I could add `:where` to make it 0-3-0 if you want, or we could leave it like this